src/modules/myMailer/myMailer.service.ts

```diff
@@ -30,15 +31,15 @@ export class MyMailerService {
           to: subscriber.email,
           from: 'Andrii',
           subject: 'Current USD to UAH rate',
-          text: `Hello my dear subscriber!\nYou can buy 1 UAH for ${usdUahRate} USD, or 1 USD for ${1 / usdUahRate} UAH.\nHave a nice day!`,
+          text: `Hello my dear subscriber!\nYou can buy 1 UAH for ${1 / usdUahRate} USD, or 1 USD for ${usdUahRate} UAH.\nHave a nice day!`,
```

Comment:
Looks like an opportunity for moving this logic to a separate function that would return this string